cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -38,6 +39,11 @@ import (
 	provider_gce "k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
 )
 
+// TODO(krzysztof-jastrzebski): Move to main.go.
+var (
+	gkeAPIEndpoint = flag.String("gke-api-endpoint", "", "GKE API endpoint address.")
```

Comment:
Can you say in comment that this is used by developers, so that it's obvious to users they shouldn't set this?

Reply:
Done